Editor/TextProvider.cs

```diff
@@ -182,6 +182,33 @@ internal sealed class TextProvider
             { Strings.TabAnywhere, "GameLift Anywhere" },
             { Strings.TabEC2, "Managed EC2" },
             { Strings.TabHelp, "Learning Resources" },
+            
+            { Strings.LabelAccountTitle, "Manage Your User Profiles"},
+            { Strings.LabelAccountDescription, "Create a profile to link to an AWS account and store your security credentials. Your profile also specifies the AWS Region you want to work in.\nYou can have multiple profiles, but only one can be active at a time. Check your active profile selection on the main page of the Amazon GameLift window."},
+            { Strings.LabelAccountCardNoAccountTitle, "I need a new AWS account for this project"},
+            { Strings.LabelAccountCardNoAccountDescription, "You need an AWS account to work with AWS services. Create a new account, then set up an account user and get security credentials."},
+            { Strings.LabelAccountCardNoAccountDescriptionLink, "Learn More."},
+            { Strings.ButtonAccountCardNoAccount, "Create an AWS Account"},
+            { Strings.LabelAccountHasAccountTitle, "I have an AWS account for this project"},
```

Comment:
minor: How about `Existing` instead of `Has` for these variables. So `LabelAccountExistingAccountTitle`?  Thoughts?